### DISCUSSION TITLE
VDH not working in Firefox

### DISCUSSION THREAD

**Discussion by arashuno** [aclap-dev/video-downloadhelper#1219] [Help and Q&A]
### Your question

Hi! Have a paid license and VDH isn't recognizing videos being played with the 130.0 Firefox update.

<img width="1728" alt="Screenshot 2024-09-05 at 11 08 58 AM" src="https://github.com/user-attachments/assets/3c411cd7-f8e1-44b0-af90-113b1e063f60">

Any direction would be appreciated. Details pasted. Thanks!

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"6dafd2f591c54a3c"}
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:130.0) Gecko/20100101 Firefox/130.0
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "/Users/shammydee/Downloads",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "/Users/shammydee/Downloads"
        }
    },
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 5,
    "successfull_dl": 75
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

I am running Windows 11 Pro 64-bit version 23H2 build 22631.4112, Firefox 130.0 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.

![#01](https://github.com/user-attachments/assets/26d0a859-ab5c-44f4-b6b9-b28227f45c7e)
![#02](https://github.com/user-attachments/assets/10fe577a-126e-4c1f-b075-3e49ad392274)
![#03](https://github.com/user-attachments/assets/c5dda2c5-dd16-4810-8628-8599ac9914e7)
![#04](https://github.com/user-attachments/assets/ea0b08f5-b115-4740-8607-d3c7e7cc6fde)

No, I cannot confirm your claim.  Everything works fine for me.

**Reply by GrampaWildWilly** [reply]
I don't believe the particular release/build of Windows is significant.  I don't believe the version of Firefox is significant.  I don't believe licensed vs unlicensed is significant.  I don't believe CoApp 2.0.19 vs 2.0.20 is significant.

I believe VDH general availability vs VDH beta is significant.  Everybody on Firefox should always be running the beta.  The beta contains bug fixes.  It is not, as is true perhaps of other software, some experimental version full of bugs & prone to errors.  It is just an early release of bug fixes.  That's why everybody on Firefox should always be running the beta.

**Comment by Mattmjs** [comment]
I see detection of this in the stable extension, I also changed the user agent to Mac 10.15 and still detection.
Maybe it's a cache & cookies issue.
